wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/wsnext/model/Container.java

```diff
@@ -0,0 +1,156 @@
+/*
+ * Copyright (c) 2012-2018 Red Hat, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+package org.eclipse.che.api.workspace.server.wsnext.model;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+public class Container {
```

Comment:
Wouldn't it be more consistent to call it `CheContainer` (as `CheService` or `CheFeature`)? In the issue where the model is described it is called `Container` but that's weird. WDYT?

Reply:
Maybe. I reused already generated files by Swagger model because I don't want to bring new changes that I would have to redo after regeneration of the model classes.
So, I believe, it is rather a suggestion for @skabashnyuk